### PR TITLE
Fix alt.unloadYtyp & alt.loadYtyp example

### PR DESCRIPTION
**Pull request by C0kkie**

### CODE REVIEW

File: client/index.d.ts [hunk 1]
```diff
@@ -1904,7 +1904,8 @@ declare module "alt-client" {
    * Load a specific ytyp file.
    *
    * @param path Relative path to the game folder.
-   * @remarks Example: alt.loadYtyp("x64u.rpf/levels/gta5/_hills/country_06/country_06_metadata.rpf/cs6_08_interior_cs6_08_mine_int.ytyp")
+   * @example
+   * alt.loadYtyp("x64u.rpf/levels/gta5/_hills/country_06/country_06_metadata.rpf/cs6_08_interior_cs6_08_mine_int.ytyp")
```
Comment (LeonMrBonnie) [inline client/index.d.ts:1908]: Change it so it is like this:
```
```js
code
```
```

File: client/index.d.ts [hunk 2]
```diff
@@ -1914,7 +1915,8 @@ declare module "alt-client" {
    * Unload a specific ytyp file.
    *
    * @param path Relative path to the game folder.
-   * @remarks Example: alt.unloadYtyp("x64u.rpf/levels/gta5/_hills/country_06/country_06_metadata.rpf/cs6_08_interior_cs6_08_mine_int.ytyp")
+   * @example
+   * alt.unloadYtyp("x64u.rpf/levels/gta5/_hills/country_06/country_06_metadata.rpf/cs6_08_interior_cs6_08_mine_int.ytyp")
```
Comment (LeonMrBonnie) [inline client/index.d.ts:1919]: Change so it is like this:
http://puu.sh/Halx1/4ed578312a.png

Comment (C0kkie) [reply client/index.d.ts:1922]: Done

**Review by Lhoerion — COMMENTED** [review]
Thanks for fix

**Pull request merged**